src/appendices/contributors.md

```diff
@@ -257,6 +257,7 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 | Mateusz Pawlik                                       | 📖🐛🤔🚧👀                             |
 | Mathias Goncalves                                    | 💻🔧📢                                 |
 | Mathieu Boudreau                                     | 💬🤔📢                                 |
+| Matt Cieslak                                         | 📖                                     |
```

Comment:
Please update https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors instead of directly modifying this.